doc/lua_api.md

```diff
@@ -274,7 +274,7 @@ Accepted formats are:
 
     images: .png, .jpg, .tga, (deprecated:) .bmp
     sounds: .ogg vorbis
-    models: .x, .b3d, .obj, .gltf (Minetest 5.10 or newer)
+    models: .x, .b3d, .obj, .gltf, .glb (Minetest 5.10 or newer)
```

Comment:
it's ambigous what model formats `(Minetest 5.10 or newer)` refers to, putting it before with a colon like above would be better

Reply:
of course, done